guide/popular-topics/webhooks.md

```diff
@@ -43,10 +43,14 @@ You can fetch a specific webhook using its `id` with <DocsLink path="class/Clien
 
 #### Using the WebhookClient constructor
 
-If you are not using a bot client, you can get a webhook by creating a new instance of `WebhookClient` and passing the `id` and `token` into the constructor. These credentials do not require you to have a bot application, but it also offers limited information instead of fetching it using an authorized client.
+If you are not using a bot client, you can get a webhook by creating a new instance of `WebhookClient` and passing the `id` and `token` or just the `url` into the constructor. These credentials do not require you to have a bot application, but it also offers limited information instead of fetching it using an authorized client.
```

Comment:
```suggestion
If you are not using a bot client, you can get a webhook by creating a new instance of `WebhookClient` and passing the `id` and `token` into the constructor. These credentials do not require you to have a bot application, but it also offers limited information instead of fetching it using an authorized client.
```